templates/disco.twig

```diff
@@ -30,7 +30,8 @@
         <ul class="tabset_tabs">
         {% for tab, idps in idplist %}
             {% if idps is not empty %}
-            <li class="tab-link{% if loop.first %}current{% endif %}" data-tab="{{ tab }}"><a href="#{{ tab }}"><span>{{ tabNames[tab]|trans }}</span></a></li>
+            {# NOTE: pass through raw to allow HTML in tabNames, e.g. Belgi&euml; #}
```

Comment:
Allowing html here is a potential security risk..
How are you defining the tab names in the config? You should be to just use a ë there

Reply:
I understand your points, although I don't think there is a risk in this case: the input is from a local config file, not from user input. Also, I'm not a big fan of hardcoding UTF-8 chars in HTML if it's not absolutely necessary (I've seen to many weird characters appearing in my surname), but I'm aware you're already doing so for other languages.
In short it's fine with me to remove this commit and only keep the other.

Reply:
They are from a config file. So I would say the risk is perfectly acceptable?

Reply:
OK, I'll merge it then